tools/dependencies/known-oap-backend-dependencies-es7.txt

```diff
@@ -143,6 +143,7 @@ parent-join-client-7.0.0.jar
 perfmark-api-0.19.0.jar
 proto-google-common-protos-1.12.0.jar
 protobuf-java-3.4.0.jar
+protobuf-java-3.11.4.jar
```

Comment:
Two versions of protobuf-java?  Please check whether the older version is still in use or not, if not, remove it here and update the file below, otherwise consider making the versions consistent

https://github.com/apache/skywalking/blob/29da5738bcea2a55e5d88f4eda9ea9056be38867/dist-material/release-docs/LICENSE#L367-L368

Reply:
The old version is still in use. The new `protobuf-java-3.11.4.jar` just used for `protobuf-java-util -3.11.4.jar`, and I used `protobuf-java-util -3.11.4.jar` to dealing with enum type as I mentioned above.